documentation/commands/git-town.md

```diff
@@ -12,6 +12,7 @@ git town hack-push-flag [(true | false)]
 git town help
 git town install-fish-autocompletion
 git town main-branch [<branch_name>]
+git town parent-branch <child_branch_name> <parent_branch_name>
```

Comment:
I still see `parent-branch` in this file. Did you forget to update this documentation?